lib/tokenize.cpp

```diff
@@ -80,7 +79,7 @@ static bool isEnumStart(const Token* tok)
 {
     if (!tok || tok->str() != "{")
         return false;
-    return (tok->strAt(-1) == "enum") || (tok->strAt(-2) == "enum") || Token::Match(tok->tokAt(-3), "enum class %name%");
+    return (tok->strAt(-1) == "enum") || (tok->strAt(-2) == "enum") || Token::Match(tok->tokAt(-3), "enum class %name%") || Token::Match(tok->tokAt(-3), "enum %type% %name%");
```

Comment:
I suggest moving the code from `isEnumScope()` here and replacing the call to `isEnumScope()` with `isEnumStart()`.